driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/access.bpf.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+#include <helpers/interfaces/variable_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(access_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct ringbuf_struct ringbuf;
+        if(!ringbuf__reserve_space(&ringbuf, ACCESS_E_SIZE))
+        {
+                return 0;
+        }
+
+        ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_ACCESS_E);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: mode (type: PT_UINT32) */
+	unsigned long mode = (u32)extract__syscall_argument(regs, 1);
```

Comment:
```suggestion
	int mode = (int)extract__syscall_argument(regs, 1);
```

Reply:
in the man page https://man7.org/linux/man-pages/man2/access.2.html `mode` is an int  so let's keep it as it is